packages/lib-classifier/src/plugins/drawingTools/components/DragHandle/DragHandle.js

```diff
@@ -2,26 +2,26 @@ import { forwardRef } from 'react';
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import draggable from '../draggable'
+import { STROKE_WIDTH } from '../Mark/Mark';
 
 const StyledCircle = styled('circle')`
   stroke-width: 2;
   &:hover {
     cursor: move;
   }
 `
-const RADIUS = screen.width > 900 ? 4 : 10
+const RADIUS = screen.width > 900 ? 3 * STROKE_WIDTH : 5 * STROKE_WIDTH
```

Comment:
I thought it would be safer to make the drag handle size a multiple of the base stroke width here. If the stroke width changes, the default handle size will change proportionately.

Reply:
Yeah, 4px seems a bit small to me, so I bumped it to 6.